docs/technology/glossary.md

```diff
@@ -0,0 +1,49 @@
+---
+id: glossary
+title: Glossary
+---
+
+**Metachain**: the blockchain that runs in a special shard, where the main responsibilities are not
+processing transactions, but notarizing and finalizing the processed shard block headers.
+
+**Address**: the public key of a wallet. The Elrond Address format is bech32, specified by the BIP 0173. The public key always starts with an `erd1`.
+e.g.: `erd1sea63y47u569ns3x5mqjf4vnygn9whkk7p6ry4rfpqyd6rd5addqyd9lf2`.
+
+**Node**: a computer or server, running the Elrond client and relaying messages received from its
+peers.
+
+**Validator**: a node on the Elrond network that staked at least 2500 EGLD, that processes
+transactions and secures the network by participating in the consensus mechanism, while
+earning rewards from the protocol and transaction fees.
+
+**Observer**: a passive member of the network that can act as a read & relay interface.
+
+**Validate**: the act of running a validator node and contributing to the network by relaying and
+validating information.
+
+**Stake**: locking funds in a staking pool in order to secure the network and receive rewards.
+
+**Delegate**: locking the funds in the “legacy” delegation contract in order to secure the network
```

Comment:
replace
locking the funds in the “legacy” delegation contract in order to secure the network

with
contribute to the network security by delegating 1 EGLD or more towards a staking provider that operates validator nodes.

Reply:
done

Reply:
the idea behing this was the "Delegate"-Tab in the wallet. There are 3 tabs there: stake (staking providers), delegate (legacy) and validate (run nodes).

-> so maybe change to: delegating 10 EGLD or more towards the Elrond Community Delegation contract community

Reply:
done. also updated the `Stake` part